pages/about.html

```diff
@@ -1,12 +1,63 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles/about_styles.css">
 </head>
+
 <body>
-  
+    <section class="container">
+        <div id="purple_1_header">
+            <header>
+                <h1>Ashley's Highlights </h1>
+                <nav>
+                    <a href="http://127.0.0.1:5500/pages/index.html" id="home"> HOME</a>
+                    <a href="http://127.0.0.1:5500/pages/portfolio.html" id="portfolio"> PORTFOLIO</a>
+                </nav>
+            </header>
+
+
+        </div>
+        <div id="purple_2_left"></div>
+        <main>
+            <section>
+                <h2>What I Like</h2>
+                <ul class="tags">
+                    <li>SQL</li>
+                    <li>TDD</li>
+                    <li>CSS</li>
+                    <li>API's</li>
+                </ul>
+            </section>
+            <br>
```

Comment:
Avoid using `br` solely for adding spacing, which can be accomplished instead with a `margin-bottom`.